monkeys/apple-tree/kubernetes-resources-charts/templates/client.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
```

Comment:
This file seems very similar to `monkeys/apple-tree/kubernetes-resources/client.yaml` which is also committed in this PR. Is there a meaningful difference or is one of these supposed to be generated from the other?